torch/_subclasses/meta_utils.py

```diff
@@ -1303,6 +1305,9 @@ def _to_fake_tensor(t: MetaTensorDesc):
                         symbolic_context=base_symbolic_context,
                     )
 
+                    if t.base.is_parameter:
+                        base._is_param = True
```

Comment:
Right now, Parameter-ification happens in the top level `__call__`

```
        if type(t) is torch.nn.Parameter:
            # NB: Cannot directly use Parameter constructor
            # because that would force a detach, not desirable
            r._is_param = True
```

If we need to recursively preserve parameter-ness (which... seems reasonable? I guess? I forgot why we preserved parameter-ness in the first place), it would seem more logical for the `_is_param` set to happen in the vicinity of `           self.set_tensor_memo(t, r)` in the main loop

Reply:
I can move it near the 'self.set_tensor_memo' ..